README.md

```diff
@@ -85,19 +90,32 @@ successful.
 
 ### patches.yml
 
-`patches.yml` is a YAML 1.2 file. It should be a list of changeset
-hashes.
+`patches.yml` is a YAML 1.2 file. It should be a list of changeset hashes and
+branch subscriptions:
 
-The hashes can refer to any changeset in any of the referenced sources.
-Each changeset will be cherry-picked, in order, to the head of the
-branch to be built.
+    superwidget:
+      changesets:
+      - {changeset1}
+      - {changeset2}
+      subscriptions:
+          - branch: remote01/widget_feature
```

Comment:
To what does "remote01" refer here?

We don't currently have a clear way to refer to extra branches; at the moment, everything assumes you're tracking the origin. I think we might need a way to talk about extra repos by name.

Reply:
I've made a change to allow users to give names to the various extras in the repo config.